### PR TITLE
fix: [SRM-9500]: Fix the margins and padding in the verification summ…

### PR DESCRIPTION
**Pull request by vasuki-kalluri**
…ary.

##### Screenshots:

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by vasuki-kalluri** [comment]
retrigger jest

File: src/modules/85-cv/components/ExecutionVerification/components/InterruptedHistory/InterruptedHistory.tsx [hunk 1]
```diff
@@ -23,11 +24,11 @@ export default function InterruptedHistory(props: InterruptedHistory): JSX.Eleme
...
   if (interruptedHistory) {
     return (
-      <Container padding={'small'} background={Color.YELLOW_100}>
-        <Text font={{ weight: 'bold' }}>
+      <Container padding={'small'} border={{ radius: 0 }} className={styles.main} background={Color.YELLOW_100}>
+        <Text font={{ weight: 'bold' }} color={Color.BLACK}>
           {getString('pipeline.failureStrategies.strategiesLabel.ManualIntervention')}
         </Text>
-        <Text>{`Set to ${interruptedHistory?.interruptType} on ${moment(
+        <Text color={Color.BLACK} font={{ size: 'small' }}>{`Set to ${interruptedHistory?.interruptType} on ${moment(
```
Comment (shashidhargg) [inline src/modules/85-cv/components/ExecutionVerification/components/InterruptedHistory/InterruptedHistory.tsx:31]: Can you please move this text to the string file?

File: src/modules/70-pipeline/components/ExecutionStepDetails/ExecutionStepDetails.module.scss [hunk 2]
```diff
@@ -15,6 +15,8 @@
 
   .header {
     padding: var(--spacing-large);
+    padding-bottom: 16px;
```
Comment (shashidhargg) [inline src/modules/70-pipeline/components/ExecutionStepDetails/ExecutionStepDetails.module.scss:18]: Can we use `var(--spacing-medium)` (15px) for 16px padding? Just 1px diff.

**Review by shashidhargg — APPROVED** [review]
LGTM

File: src/modules/70-pipeline/components/ExecutionStepDetails/ExecutionStepDetails.module.scss [hunk 3]
```diff
@@ -15,6 +15,8 @@
 
   .header {
     padding: var(--spacing-large);
```
Comment (vkbansal) [inline src/modules/70-pipeline/components/ExecutionStepDetails/ExecutionStepDetails.module.scss:17]:
```suggestion
    padding: var(--spacing-medium) var(--spacing-large);
```

File: src/modules/70-pipeline/components/ExecutionStepDetails/ExecutionStepDetails.module.scss (hunk 2 above)
Comment (vkbansal) [reply src/modules/70-pipeline/components/ExecutionStepDetails/ExecutionStepDetails.module.scss:18]: remove these lines and fix as suggested above

File: src/modules/85-cv/components/ExecutionVerification/components/InterruptedHistory/InterruptedHistory.tsx (hunk 1 above)
Comment (vkbansal) [reply src/modules/85-cv/components/ExecutionVerification/components/InterruptedHistory/InterruptedHistory.tsx:31]: +1

File: src/modules/70-pipeline/components/ExecutionStepDetails/ExecutionStepDetails.module.scss (hunk 2 above)
Comment (vasuki-kalluri) [reply src/modules/70-pipeline/components/ExecutionStepDetails/ExecutionStepDetails.module.scss:18]: PO was specific about using 16 pixels.

File: src/modules/85-cv/components/ExecutionVerification/components/InterruptedHistory/InterruptedHistory.tsx (hunk 1 above)
Comment (vasuki-kalluri) [reply src/modules/85-cv/components/ExecutionVerification/components/InterruptedHistory/InterruptedHistory.tsx:31]: This was already existing. I just added color and font.

File: src/modules/70-pipeline/components/ExecutionStepDetails/ExecutionStepDetails.module.scss (hunk 3 above)
Comment (YounixIO) [reply src/modules/70-pipeline/components/ExecutionStepDetails/ExecutionStepDetails.module.scss:17]: +1

File: src/modules/85-cv/components/ExecutionVerification/components/ExecutionVerificationSummary/components/SummaryOfDeployedNodes/SummaryOfDeployedNodes.module.scss [hunk 4]
```diff
@@ -6,12 +6,13 @@
...
 .main {
-  margin-top: 16px !important;
+  margin-top: 30px !important;
+  margin-left: 16px !important;
   .summaryContainer {
     width: 80% !important;
     margin: 30px auto !important;
     margin-top: 16px !important;
-    margin-left: 0 !important;
+    margin-left: 0px !important;
```
Comment (YounixIO) [inline src/modules/85-cv/components/ExecutionVerification/components/ExecutionVerificationSummary/components/SummaryOfDeployedNodes/SummaryOfDeployedNodes.module.scss:15]: use varibles

File: src/modules/70-pipeline/components/ExecutionStepDetails/ExecutionStepDetails.module.scss (hunk 2 above)
Comment (vasuki-kalluri) [reply src/modules/70-pipeline/components/ExecutionStepDetails/ExecutionStepDetails.module.scss:18]: Done

File: src/modules/70-pipeline/components/ExecutionStepDetails/ExecutionStepDetails.module.scss [hunk 5]
```diff
@@ -15,6 +15,8 @@
...
+    padding-bottom: var(--spacing-medium);
+    padding-top: var(--spacing-medium);
```
Comment (ishantmehtaharness) [inline src/modules/70-pipeline/components/ExecutionStepDetails/ExecutionStepDetails.module.scss:19]: We can write this in single line as @vkbansal suggested
`padding: var(--spacing-medium) var(--spacing-large);`

**Review by ishantmehtaharness — APPROVED** [review]
Looks Good
1 comment
https://github.com/harness/harness-core-ui/pull/7948/files#r832894052

File: src/modules/70-pipeline/components/ExecutionStepDetails/ExecutionStepDetails.module.scss (hunk 3 above)
Comment (vasuki-kalluri) [reply src/modules/70-pipeline/components/ExecutionStepDetails/ExecutionStepDetails.module.scss:17]: image
padding and padding-bottom are different variables.

**Review by shashidhargg — APPROVED** [review]

**Pull request merged**